ItemIndianTea.prefab

```diff
@@ -0,0 +1,22 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:items#IndianTea",
```

Comment:
This is the address for the icon, but the format `items#IndianTea` means to look for a texture atlas called `items.png` then an entry in the atlas defined as `IndianTea` - you'll want to take off `items#` and rename / re-export your image to `IndianTea.png` :-)